src/main/java/com/cloudbees/plugins/deployer/DeployNowRunAction.java

```diff
@@ -397,11 +396,7 @@ public InputStream getLogInputStream() throws IOException {
     }
 
     public Reader getLogReader() throws IOException {
-        if (getCharset() == null) {
-            return new InputStreamReader(getLogInputStream());
-        } else {
-            return new InputStreamReader(getLogInputStream(), getCharset());
-        }
+        return new InputStreamReader(getLogInputStream(), getCharset());
```

Comment:
getCharset() is already checking for nullability and returning default charset in that case.